gen/dcos-config.yaml

```diff
@@ -1826,6 +1842,12 @@ package:
               {
                   "Command": ["iptables-save"]
               },
+              {
+                  "Command": ["ip6tables-save"]
```

Comment:
Please add a test for this

Reply:
https://github.com/dcos/dcos/pull/5170

I'm curious, why didn't tests fail on this PR?

Reply:
@janisz the test has passed in #5170, but how can we be sure that the issue is fixed if it was not failing consistently before?